cocos/2d/assets/sprite-frame.ts

```diff
@@ -500,13 +506,21 @@ export class SpriteFrame extends Asset {
         this._calculateUV();
     }
 
+    /**
+     * @en Sets whether sprite can be packed into dynamic atlas.
+     * @zh 设置精灵是否允许参与自动合图。
+     */
     get packable () {
         return this._packable;
     }
     set packable (value: boolean) {
         this._packable = value;
     }
 
+    /**
+     * @en Original info before packed to dynamic atlas,includes texture and width and height.Before packed to dynamic atlas is null.
```

Comment:
info -> information. Before packed to dynamic atlas is null. -> it's null before being packed to dynamic atlas.